starknet-providers/src/jsonrpc/models/codegen.rs

```diff
@@ -38,7 +38,7 @@ pub struct ResultPageRequest {
 ///
 /// Event information decorated with metadata on where it was emitted.
 #[serde_as]
-#[derive(Debug, Clone, Serialize, Deserialize)]
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
```

Comment:
As noted on the first line of this file, this file is automatically generated and thus should not be manually edited, as any changes will be overwritten subsequently.